LambdaFactory/Fornax
# Port Fornax to .NET Core and make it work as a global dotnet tool

It works, pretty much all that's left is to fix all the build stuff like updating FAKE.  I could really use some help with that, since I don't use FAKE in my day-to-day.  And actually I'm not really sure of the value it adds here since we're not really doing anything complicated (but am happy to have it explained to me).  Maybe if we make it take care of deploying to Nuget, Brew, etc. it would make more sense.

To test:
1. `git clone git@github.com:NatElkins/Fornax.git`
2. `cd Fornax`
3. `git checkout netcore`
4. `cd src/Fornax`
5. `./resetTool.sh`.  You may need to run `chmod +x resetTool.sh`.  This is a little script I use when I want to test the tool.  It just uninstalls Fornax as a global dotnet tool, and reinstalls it.
6. `mkdir test && cd test`
7. `fornax new`
8. `fornax watch`.  If you don't have the Sass compiler installed you'll get an error since the default template uses .scss files.  You can install it here (and the error message has been changed to include this link too): https://sass-lang.com/install
9. Navigate to `localhost:8080`.

Like I said, the last thing I could use help with is figuring out the FAKE/build stuff.  Also just a heads up, I changed the way the default template for `fornax new` is being included, you can see the I'm using `<Content Include=...` in Fornax.fsproj.

I have a ton of other changes I'd like to make to this tool, but for now I just want to reach feature parity with the tool as it is published now.  Then maybe we can get into things like themes, support for lists of items (i.e. `/posts/` would call the right template to render a list of posts), and all the other goodies that other static site generators have.
